bundles/WebToPrintBundle/.github/ci/files/config/bundles.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    "Pimcore\\Bundle\\WebToPrintBundle\\PimcoreWebToPrintBundle" => ['all' => true],
```

Comment:
Is this really necessary?

Reply:
since tests are there, not necessary.

Should i remove these https://github.com/pimcore/pimcore/blob/web2print_bundle_prep/bundles/WebToPrintBundle/codeception.dist.yml and let them to be added once and if tests would be provided on the bundle repository itself?

Reply:
Yeah, I'd re-add the stuff needed when implementing the tests... just to ensure we're not having unnecessary stuff in the repo.